java/src/org/openqa/selenium/chromium/AddHasCdp.java

```diff
@@ -0,0 +1,69 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.chromium;
+
+import com.google.common.collect.ImmutableMap;
+import org.openqa.selenium.Capabilities;
+import org.openqa.selenium.internal.Require;
+import org.openqa.selenium.remote.AdditionalHttpCommands;
+import org.openqa.selenium.remote.AugmenterProvider;
+import org.openqa.selenium.remote.CommandInfo;
+import org.openqa.selenium.remote.ExecuteMethod;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Predicate;
+
+import static org.openqa.selenium.remote.BrowserType.CHROME;
+import static org.openqa.selenium.remote.BrowserType.EDGE;
+
+public abstract class AddHasCdp implements AugmenterProvider<HasCdp>, AdditionalHttpCommands {
+
+    public static final String EXECUTE_CDP = "executeCdpCommand";
+
+  @Override
+  public abstract Map<String, CommandInfo> getAdditionalCommands();
+
+  @Override
+  public Predicate<Capabilities> isApplicable() {
+    String[] validBrowsers = new String[] { EDGE, CHROME, "msedge" };
+    return caps -> new ArrayList<>(Arrays.asList(validBrowsers)).contains(caps.getBrowserName());
```

Comment:
Because creating ephemeral lists just for a contains causes churn on the garbage collector (and, you know, the list is constant)